src/libs/cropOrRotateImage/utils.ts

```diff
@@ -0,0 +1,16 @@
+import {SaveFormat} from 'expo-image-manipulator';
+
+function getSaveFormat(type: string) {
+    switch (type) {
```

Comment:
Currently we support: `jpg, jpeg, png, gif, bmp, svg`. For `jpg, bmp, gif` the fallback as `jpeg` is fine, but if I upload an `svg` today on staging/production,  
the fallback is png. With this will it be jpeg?

Reply:
It's being [handled here](https://github.com/Expensify/App/blob/main/src/components/AvatarCropModal/AvatarCropModal.tsx#L289-L295), I just changed the library, but I tried to keep logic untouched.